async_upnp_client/profiles/dlna.py

```diff
@@ -223,6 +225,12 @@ def split_commas(input_: str) -> List[str]:
     return [item for item in stripped if item]
 
 
+@lru_cache(maxsize=128)
+def _lower_split_commas(input_: str) -> Set[str]:
```

Comment:
Not that it matters much, as the size of the resulting collection is usually small... Wouldn't a `List` be faster than a `Set`? A `Set` ensures no duplicates, thus testing if the same element is not already contained.

Reply:
A `List` would be faster to construct but everything consuming this is searching it for a value so the `Set` is much faster to consume since it avoids the linear searching, and since its cached the cost to build it is only incurred infrequently.